ntp-proto/src/peer.rs

```diff
@@ -264,7 +281,7 @@ pub enum AcceptSynchronizationError {
 #[allow(clippy::large_enum_variant)]
 pub enum Update {
     BareUpdate(PeerSnapshot),
-    NewMeasurement(PeerSnapshot, Measurement, NtpPacket<'static>),
+    NewMeasurement(PeerSnapshot, Measurement),
```

Comment:
maybe this clippy allow can now be removed?

Reply:
Good catch, also removed the clippy allow on `MsgForSystem`